resources/assets/js/components/ProgressRing.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React from 'react';
```

Comment:
It appears you have different autoformat settings that I do. We should all be using the same config, and it should be using double-quotes. Lets look into this.